### PR TITLE
adding build script for tempo-operator image

### PR DESCRIPTION
**Pull request by anurag-chitrakar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Have you validated script on UBI 9 container
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by anurag-chitrakar** [comment]
logs for tempo-operator 
[logs_tempo_operator_rhel_9.3.txt](https://github.com/user-attachments/files/16192335/logs_tempo_operator_rhel_9.3.txt)
@prasanna-marathe @Swapnali911

**Comment by prasanna-marathe** [comment]
Travis build is failed . Please look into this

**Comment by prasanna-marathe** [comment]
Building master branch. Please check [CONTRIBUTING.md](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md) and update code accordingly

File: t/tempo-operator/tempo-operator_rhel_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export GO_VERSION=$(cat go.mod | grep go | head -n 1 | cut -d " " -f2)
+
+# Install go 1.22.0
+
+export PATH=$PATH:/usr/local/go/bin
+export GOPATH=/root/go
+export GOBIN=/usr/local/go/bin
+if [ $( go version | cut -d " " -f3 ) = "go$GO_VERSION" ]; then
+    echo "$GO_VERSION is already installed"
+else
```
Comment (Swapnali911) [inline t/tempo-operator/tempo-operator_rhel_9.3.sh:45]: this check is not handling the use-case when GO_VERSION  is different then installed go version

Comment (anurag-chitrakar) [reply t/tempo-operator/tempo-operator_rhel_9.3.sh:45]: resolving that

Comment (anurag-chitrakar) [reply t/tempo-operator/tempo-operator_rhel_9.3.sh]: logs for tempo-operator 
[tempo-operator-ubi9-logs.txt](https://github.com/user-attachments/files/16310593/tempo-operator-ubi9-logs.txt)

File: t/tempo-operator/tempo-operator_rhel_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=tempo-operator
+PACKAGE_URL=https://github.com/grafana/tempo-operator
+PACKAGE_VERSION=${1:-main}
```
[the remark this answers is not in the document]
Comment (anurag-chitrakar) [reply t/tempo-operator/tempo-operator_rhel_9.3.sh]: logs for tempo-operator
[tempo-operator-ubi9-logs.txt](https://github.com/user-attachments/files/16310616/tempo-operator-ubi9-logs.txt)

**Review by prasanna-marathe — APPROVED** [review]

**Pull request merged**